openapi.yaml

```diff
@@ -0,0 +1,698 @@
+swagger: '2.0'
+info:
+  title: 'Game Curator API'
+  description: 'Learning project API to retrieve and manipulate game review data.'
+  version: 'v1'
+  license:
+    name: GNU Affero General Public License Version 3
+    url: http://www.gnu.org/licenses/agpl-3.0.en.html
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+schemes:
+  - http
+host: 'api.oregonstate.edu'
+basePath: /v1
+externalDocs:
+  description: GitHub Repository
+  url: https://github.com/osu-mist/game-curator-api
+produces:
+  - application/json
+paths:
+  /games:
+    parameters:
+      - $ref: '#/parameters/authorization'
+    get:
+      summary: List all games
+      description: Get all games
```

Comment:
You should as add `operationId` fields for all of your methods. Example: https://github.com/osu-mist/express-api-skeleton/blob/master/openapi.yaml#L29

Reply:
Added operationIds to all methods